cli/internal/fs/turbo_json.go

```diff
@@ -551,11 +564,15 @@ func (c *TurboJSON) UnmarshalJSON(data []byte) error {
 		envVarDependencies.Add(value)
 	}
 
+	// TODO: In the rust port, warnings should be refactored to a post-parse validation step
```

Comment:
For what it's worth we have a `TurboJSON.Validate()` method that can do this now. Since this isn't throwing an error though, it would have to be adapted slightly to emit warnings rather than errors.